fixed get started section less padding

kayliezhao opened the pull request:
fixed the photo component for sizing issues , added the get started section
<img width="934" alt="image" src="https://github.com/user-attachments/assets/caad8b0f-271d-480d-b64c-98dff93e5d17">

kayliezhao committed:
fixed photo component for felxibility, added get started section

SafeDuck commented on src/components/Photo.tsx:
```diff
@@ -1,22 +1,28 @@
   // typescript-react
-  image: string;
+  image: string | StaticImageData;
   width: number;
```
let's make width a string. In particular a tailwind class. 
For example `w-1/2` or `w-1/4`

SafeDuck commented on src/components/Photo.tsx:
```diff
@@ -1,22 +1,28 @@
       <Image
         src={image}
         alt={alt}
         width={width}
```
Remove the `width` prop from the Image component.
Use a tailwind class instead, aka the string `width` from the props

SafeDuck commented on src/components/home/GetStarted.tsx:
```diff
@@ -1,9 +1,44 @@
+              Waiver: ***Highlander Gloves Waiver Form (must sign before first
+              practice!)
+            </div>
+            <div className="ml-11 flex w-[200%] translate-x-[-30%] justify-center p-11">
```
the `translate` shouldn't be needed. This component can be done with `flex` alone

SafeDuck commented on src/components/home/GetStarted.tsx:
```diff
@@ -1,9 +1,44 @@
+              />
+            </div>
+          </div>
+          <div className="ml-11 pr-32">
```
Too much hard coding padding. 
Use `flex` properties instead. Refer to example:
![image](https://github.com/user-attachments/assets/387d6339-476a-4d94-b06d-2cacfad93400)

Essentially, we want less on padding and more reliance on `flex` alone

kayliezhao committed:
Merge branch 'dev' of https://github.com/acm-ucr/highlander-gloves-website into kayliezhao/getStarted

kayliezhao committed:
used less padding, adjusted the button component so i dont have to double the size and translate it to center it

kayliezhao commented:
removed width prop, used less padding, adjusted button component so i dont have to translate when i resize the button width
<img width="899" alt="image" src="https://github.com/user-attachments/assets/d4b42b3d-02d2-41b5-aeba-ebaa51fea19f">

SafeDuck committed:
fixes
